src/resolve/resolved-resolv-conf.c

```diff
@@ -324,6 +324,12 @@ static int write_stub_resolv_conf_contents(FILE *f, OrderedSet *dns, OrderedSet
         return fflush_and_check(f);
 }
 
+void manager_symlink_stub_to_uplink_resolv_conf(void) {
+        int r = symlink_atomic_label(basename(PRIVATE_UPLINK_RESOLV_CONF), PRIVATE_STUB_RESOLV_CONF);
```

Comment:
nitpick: we have this rule that `int r;` is special and always gets defined separately and at outermost function scope. Hence, please split this in two lines, i.e.:

```c
int r;

r = symlink…
```